.ci/travis.sh

```diff
@@ -131,7 +131,10 @@ jacoco)
     jacoco:check@default-check
   # BUILD_REASON is variable from CI, if launch is not from CI, we skip this step
   if [ -n "$BUILD_REASON" ];then
+    echo "Reporting to codecov"
     bash <(curl -s https://codecov.io/bash)
```

Comment:
https://github.com/codecov/codecov-bash/blob/master/readme.md
> On February 1, 2022, this uploader will be fully sunset and no longer function
> We recommend all users migrate to the [uploader](https://github.com/codecov/uploader) to prevent any breakages in usage.